opendevin/core/main.py

```diff
@@ -26,7 +26,10 @@ def read_task_from_stdin() -> str:
 
 
 async def main(task_str: str = ''):
```

Comment:
This change modifies the return type of this function to `None`, so just to be safe could you add typing info? 

```suggestion
async def main(task_str: str = '') -> None:
```